Include/interactivity.h

```diff
@@ -237,6 +237,13 @@ class interactive_participant
 enum class interactive_event_type
 {
     /// <summary>
+    /// The message is not a message processed by the SDK. It could be a custom
+    /// message from an interactive control or a message supported by the protocol
+    /// but not yet supported by the SDK.
+    /// </summary>
+    unknown,

```

Comment:
>unknown [](start = 4, length = 7)

not sure I like the term "unknown" - maybe "generic" or something like it? unknown sounds unsafe.